examples/trace/client.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+
+from opentelemetry import trace
+from opentelemetry.ext import http_requests
+from opentelemetry.sdk.trace import Tracer
+from opentelemetry.sdk.trace.export import (
+    ConsoleSpanExporter,
+    SimpleExportSpanProcessor,
+)
+
+trace.set_preferred_tracer_implementation(lambda T: Tracer())
+tracer = trace.tracer()
+http_requests.enable(tracer)
+span_processor = SimpleExportSpanProcessor(ConsoleSpanExporter())
+tracer.add_span_processor(span_processor)
```

Comment:
Not for this PR but: This example nicely shows that we have to improve our SDK initialization/setup experience. Ideally, I'd like this all (except for the http_requests extension) to be a single line like `opentelemetry.sdk.setup()`. The setup function would work in a similar fashion to https://docs.python.org/3/library/logging.html#logging.basicConfig

Reply:
Definitely! I've been telling people that my **actual goal** for this PR is to remind that we still have a long way to go to make it simple - currently the SDK is **hard to use**. :)

Reply:
#195